integration

```diff
@@ -781,6 +781,7 @@
   "JonasJoKuJonas/homeassistant-trias",
   "JonasJoKuJonas/homeassistant-WebUntis",
   "JonasPed/homeassistant-eloverblik",
+  "jonducrou/ha_visualiser",
```

Comment:
https://github.com/jonducrou/ha_visualiser/blob/v0.8.2/custom_components/ha_visualiser/__init__.py#L50
Overwriting PATHS used by HACS will not be allowed.
If you need, you can use this repository as custom.